.github/workflows/sca.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         operating-system:
           - ubuntu-24.04
         php-version:
-          - 8.3
+          - 8.3.8
```

Comment:
could you share why this is fixing the PHPStan ?

Reply:
I was hoping this would show the failing SCA step is because of PHP patch version change, but using patch version is not working (see https://github.com/shivammathur/setup-php/issues/618 for details), so you have to trust about this.

@Wirone's [PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8107) was run on PHP 8.3.8 and passed, PHP 8.3.9 somehow changes the lines of code for PHPStan.

CC @ondrejmirtes FYI ☝🏼

Reply:
Please don't cc me like that. If you have something to report or a question to ask, please do it properly on PHPStan's GitHub.

Reply:
Sorry, I thought you might be interested in knowing that upgrading from PHP 8.3.8 to PHP 8.3.9 may change errors reported by PHPStan.